__tests__/lib/messageUtils.test.ts

```diff
@@ -56,3 +56,31 @@ describe('toBinary', () => {
     expect(decodedResult).toEqual(nonLatinString);
   })
 })
+
+describe('getDashboard', () => {
+  // Test that get dashboard returns a correct dashboard links for a given
```

Comment:
nit: comment typos :)

Reply:
I actually just deleted this and the other comment, but they are both just repeating what is already in the `it()` clause directly below them.